halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/security/OAuth2.java

```diff
@@ -168,6 +181,7 @@ public String toString() {
     AZURE("azure"),
     GITHUB("github"),
     ORACLE("oracle"),
+    BYO("byo"),
```

Comment:
Can we name this "other" or "custom"? Simply "byo" isn't clear that it means "bring your own".